draft-demarco-oauth-status-attestations.md

```diff
@@ -258,17 +258,19 @@ to the public key attested by the Credential Issuer and contained within the Dig
 ~~~
 POST /status HTTP/1.1
 Host: issuer.example.org
-Content-Type: application/x-www-form-urlencoded
+Content-Type: application/json
 
-credential_pop=$CredentialPoPJWT
+{ "credential_pop" : [$CredentialPoPJWT, $CredentialPoPCWT, ... ] }
```

Comment:
if these need more params, we might consider this interface to be an array of objects containing encoded credentials.

Reply:
What about encoding any required parameter within the JWT/CWT according to the schema type `status-attestation+{jwt,cwt}`?

this would make each object consistent in its purpose

we may say that any additional parameter could be included, while only the required set of attributes, for each status request is defined within the specs

Reply:
therefore we could rename credential_pop to `status_assertion_requests` where each element within the array configures a `typ` value set to `status-assertion-request+{jwt,cwt}`

the corresponding response would be a json array containing for each element with a `typ` value set to `status-assertion+{jwt,cwt}` or ar `status-assertion-error+{jwt,cwt}` at the same position of the corresponding one within the request.

Reply:
You would need to verify / decode the protected header to distinguish the types... but it is nice that they are required to advertise their typ consistent with the JWT BCP.

Reply:
yes, I was wondering about "in object veritas"